include/circt/Dialect/FIRRTL/Passes.td

```diff
@@ -671,19 +671,30 @@ def LowerXMR : Pass<"firrtl-lower-xmr", "firrtl::CircuitOp"> {
   ];
 }
 
-def LowerIntrinsics : Pass<"firrtl-lower-intrinsics", "firrtl::CircuitOp"> {
-  let summary = "Lower intrinsics";
+def IntrinsicInstancesToOps : Pass<"firrtl-intrinsic-instances-to-ops", "firrtl::CircuitOp"> {
```

Comment:
Mega-bikeshed: how about `LowerIntmodules`?

Reply:
I actually meant to explicitly invite suggestions for a better name, thanks!

Reply:
Going with the suggestion, thanks!
Wasn't sure since it's not obviously "lower" to go to the generic op format, and also operates mostly on instances OF intmodule's but I think this makes sense and is a lot more ergonomic.